js/reference.js

```diff
@@ -3111,6 +3129,18 @@ var ERMrest = (function(module) {
         },
 
         /**
+         * Formats the presentation value corresponding to this reference-column definition.
+         * @param {String} data In case of pseudocolumn it's the raw data, otherwise'formatted' data value.
+         * @param {Object} options includes `context` and `formattedValues`
+         * @returns {Object} A key value pair containing value and isHTML that detemrines the presenation.
+         */
+        formatJsonPresentation: function(data, options) {
```

Comment:
What's the purpose of this function? This should be part of your `formatvalue` function. If your `formatvalue` function is different in different contexts, you can use the context that we are passing as an option to figure out which way to format the JSON column.